Library/Homebrew/formula.rb

```diff
@@ -307,6 +308,12 @@ def any_version_installed?
   end
 
   # @private
+  # If any old version of {Formula} is installed.
```

Comment:
`any_old_version_installed?` and `any_version_installed?` is two different methods.